### PR TITLE
Update oneVPL samples

### PR DESCRIPTION
**Pull request by jeffreymcallister**
# Introduction
- Changes to existing samples

# Existing Sample Changes
## Description

This PR includes multiple updates to enable use on GPU as well as to align with oneVPL spec updates.

## External Dependencies

In Linux the sample build searches for standard GPU stack dependencies.  If they are not found the samples gracefully build without GPU support.

## Type of change

Please delete options that are not relevant. Add a 'X' to the one that is applicable. 

- [ X] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [X] Command Line
- [X ] When compiling the compliler flag "-Wall -Wformat-security -Werror=format-security" was used

### CODE REVIEW

**Comment by JoeOster** [comment]
@mav-intel - Can you take a look at this as your oneVPL domain expert. With these substantial changes, I just want confirmation from you they are in line with your expectations.

**Review by JoeOster — DISMISSED** [review]
approved for ci only

**Review by JoeOster — APPROVED** [review]
approved for ci only

File: Libraries/oneVPL/dpcpp-blur/README.md [hunk 1]
```diff
@@ -32,8 +33,11 @@ If the oneAPI DPC++ Compiler is not found, the blur operation will be disabled.
 | Target device     | CPU
 | Input format      | I420
 | Output format     | BGRA raw video elementary stream
-| Output resolution | same as input
+| Output resolution | 256x192
 
+## Linux GPU optimizations
+
```
Comment (tomlenth) [inline Libraries/oneVPL/dpcpp-blur/README.md:39]: Can you explain what this does a little.  it's very out of context.  If you intend for the user to enter this command, it should be in the same code format as the commands for running the sample.

**Review by tomlenth — CHANGES_REQUESTED** [review]
Please include in the readmes the sections specific to the include file and running in the devcloud.  Examples can be found in the mandelbort readme: https://github.com/oneapi-src/oneAPI-samples/tree/master/DirectProgramming/DPC%2B%2B/CombinationalLogic/mandelbrot

File: Libraries/oneVPL/dpcpp-blur/README.md (hunk 1 above)
Comment (jeffreymcallister) [reply Libraries/oneVPL/dpcpp-blur/README.md]: Thanks for finding this -- this section was not for this version of the sample.  Removed.  For general GPU setup the user can follow the oneAPI basekit setup instructions.

File: Libraries/oneVPL/dpcpp-blur/README.md [hunk 2]
```diff
@@ -92,10 +93,10 @@ Perform the following steps:
    <oneapi_install_dir>\setvars.bat
    ```
    Here `<oneapi_install_dir>` represents the root folder of your oneAPI
-   installation, which is `C:\Program Files (x86)\Intel\oneAPI\`
+   installation, which is which is `C:\Program Files (x86)\Intel\oneAPI\`
```
Comment (mliu2020) [inline Libraries/oneVPL/dpcpp-blur/README.md:96]: Does this double "which is"?

Mark

File: Libraries/oneVPL/hello-decode/README.md [hunk 3]
```diff
@@ -87,10 +88,10 @@ Perform the following steps:
    <oneapi_install_dir>\setvars.bat
    ```
    Here `<oneapi_install_dir>` represents the root folder of your oneAPI
-   installation, which is `C:\Program Files (x86)\Intel\oneAPI\`
+   installation, which is which is `C:\Program Files (x86)\Intel\oneAPI\`
```
Comment (mliu2020) [inline Libraries/oneVPL/hello-decode/README.md:91]: "which is" is doubled.

File: Libraries/oneVPL/hello-vpp/README.md [hunk 4]
```diff
@@ -85,10 +85,10 @@ Perform the following steps:
    <oneapi_install_dir>\setvars.bat
    ```
    Here `<oneapi_install_dir>` represents the root folder of your oneAPI
-   installation, which is `C:\Program Files (x86)\Intel\oneAPI\`
+   installation, which is which is `C:\Program Files (x86)\Intel\oneAPI\`
```
Comment (mliu2020) [inline Libraries/oneVPL/hello-vpp/README.md:88]: "which is" is doubled.

File: Libraries/oneVPL/dpcpp-blur/README.md [hunk 5]
```diff
@@ -1,38 +1,36 @@
 # `dpcpp-blur` Sample
```
Comment (JoeOster) [inline Libraries/oneVPL/dpcpp-blur/README.md:1]: I made a few grammar and punctuation changes to this document.

File: Libraries/oneVPL/hello-decode/README.md [hunk 6]
```diff
@@ -1,11 +1,11 @@
 # `hello-decode` Sample
 
 This sample shows how to use the oneAPI Video Processing Library (oneVPL) to
-perform a simple video decode.
+perform a simple video decode using 2.x enhanced programming model APIs.
 
 | Optimized for    | Description
 |----------------- | ----------------------------------------
-| OS               | Ubuntu* 18.04; Windows* 10
+| OS               | Ubuntu* 20.04; Windows* 10
```
Comment (JoeOster) [inline Libraries/oneVPL/hello-decode/README.md:8]: Most samples work in 18.04 and 20.04, is there a reason for the change?

File: Libraries/oneVPL/hello-decode/README.md [hunk 7]
```diff
@@ -1,11 +1,11 @@
 # `hello-decode` Sample
```
Comment (JoeOster) [inline Libraries/oneVPL/hello-decode/README.md:1]: Please note I made some grammar and punctuation changes

**Review by JoeOster — CHANGES_REQUESTED** [review]
I altered all the readmes with minor changes, please alter if I changed the messaging to much and or let me know if you are good with the changes

File: Libraries/oneVPL/hello-decode/README.md (hunk 6 above)
Comment (jeffreymcallister) [reply Libraries/oneVPL/hello-decode/README.md:8]: Only 20.04 is supported for GPU.

**Comment by mav-intel** [comment]
> @mav-intel - Can you take a look at this as your oneVPL domain expert. With these substantial changes, I just want confirmation from you they are in line with your expectations.

Hi Joe. Yes these changes are POR and expected

**Comment by jeffreymcallister** [comment]
Accepted all changes, added additional readme cleanup, and added sections requested by Tom.  Squashed so that everything is ready to go in as one commit.

**Review by JoeOster — APPROVED** [review]

**Review by JoeOster — APPROVED** [review]

**Review by JoeOster — APPROVED** [review]
approved for ci

**Review by JoeOster — APPROVED** [review]
approved for ci

**Pull request closed without merging**